payload-files/express/test-error--transaction.json

```diff
@@ -53,27 +53,50 @@
         "name": "macOS",
         "version": "14.3"
       },
+      "otel": {
+        "resource": {
+          "service.name": "node",
+          "service.namespace": "sentry",
+          "service.version": "8.0.0-beta.2",
+          "telemetry.sdk.language": "nodejs",
+          "telemetry.sdk.name": "opentelemetry",
+          "telemetry.sdk.version": "1.23.0"
+        }
+      },
       "runtime": {
         "name": "node",
         "version": "v20.12.1"
       },
       "trace": {
         "data": {
+          "http.flavor": "1.1",
+          "http.host": "localhost:3030",
+          "http.method": "GET",
           "http.response.status_code": 200,
-          "query": {},
+          "http.route": "/test-error",
+          "http.scheme": "http",
+          "http.status_code": 200,
+          "http.status_text": "OK",
+          "http.target": "/test-error",
+          "http.url": "http://localhost:3030/test-error",
+          "http.user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36",
+          "net.host.ip": "::1",
+          "net.host.name": "localhost",
+          "net.host.port": "[[highNumber]]",
+          "net.peer.ip": "::1",
+          "net.peer.port": "[[highNumber]]",
+          "net.transport": "ip_tcp",
+          "otel.kind": "SERVER",
           "sentry.op": "http.server",
-          "sentry.origin": "auto.http.node.tracingHandler",
+          "sentry.origin": "auto.http.otel.http",
           "sentry.sample_rate": 1,
           "sentry.source": "route",
-          "url": "/test-error"
+          "url": "http://localhost:3030/test-error"
```

Comment:
`url` is now always a full URL, which is actually better than what we had before.